setup/www/tools/promote/settings

```diff
@@ -1,8 +1,14 @@
 staging_rootdir=/home/staging/${site}/
 dist_rootdir=/home/dist/${site}/
-nightly_srcdir=${staging_rootdir}nightly
-next_nightly_srcdir=${staging_rootdir}next-nightly
+
 release_srcdir=${staging_rootdir}release
+release_dstdir=${dist_rootdir}release
```

Comment:
old change adding rc builds and shuffling the order so the release types are next to eachother instead of dst grouped and src grouped